package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "my-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "^2.1.3",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
- [ ] You have a problem with dependency tree specifically with babel-jest, please check the error message it is very detailed on how you can fix it.
![image](https://user-images.githubusercontent.com/62705947/219393955-76b67c34-56b0-4d6a-b948-b26a0d75d67f.png)

Reply:
- [ ] I forgot to check correctly your Readme file. You should include all the main headings from the Microverse template. Please update your readme because otherwise the project would be invalidated.